src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -97,6 +98,10 @@
             optional = true)
     protected long bamPartitionSplitSize = 0;
 
+    @Argument(doc = "Whether to use NIO or the Hadoop filesystem (default) for reading files.",
```

Comment:
This should probably have a note that it doesn't work for writing files currently? What happens if you try to write to a bucket with use-nio mode enabled? One hopes it fails gracefully without throwing some nasty authentication error. We should probably at least have a test of that behavior for the time being.

Reply:
It would use the GCS Hadoop connector.

Reply:
Alright that makes sense, i would still add a note saying that NIO is currently not supported for writing. I would also add a test asserting that the round trip (NIO in -> GCS Out) is functioning. That shouldn't be to hard I don't think.

Reply:
Added a note saying NIO is not supported for writing, and added a test to assert the round trip is working.